src/goto-instrument/unwindset.cpp

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************\
+
+Module: Loop unwinding setup
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#include "unwindset.h"
+
+#include <util/string_utils.h>
+#include <util/string2int.h>
+
+void unwindsett::parse_unwind(const std::string &unwind)
+{
+  if(!unwind.empty())
+    global_limit=unsafe_string2unsigned(unwind);
```

Comment:
I would have gone for the `safe` variant, in particular as it is user-provided input.